### PR TITLE
SONAR: Immediately return this expression instead of assigning it to …

### PR DESCRIPTION
**Pull request by slawekk87**
…the temporary variable pt1

## Changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [ ] My code is written according to the guidelines found [here] (https://yetiforce.com/en/knowledge-base/documentation/developer-documentation/item/how-to-create-php-files).
- [ ] I have signed the Contributor Licence Agreement between me and YetiForce.

### CODE REVIEW

File: modules/OSSTimeControl/uitypes/Time.php [hunk 1]
```diff
@@ -43,8 +43,6 @@ public function getDisplayTimeDifferenceValue($fieldName, $value)
 		}
 
 		$dateTimeField = new DateTimeField($date->format('Y-m-d H:i:s'));
-		$value = $dateTimeField->getDisplayTime();
-
-		return $value;
+		return $dateTimeField->getDisplayTime();
```
Comment (TomaszQr) [inline modules/OSSTimeControl/uitypes/Time.php]: (new DateTimeField($date->format('Y-m-d H:i:s')))->getDisplayTime();

File: modules/Products/models/TreeView.php [hunk 2]
```diff
@@ -48,8 +48,6 @@ public function getTreeList()
 	{
 		$tree = parent::getTreeList();
 		$treeWithItems = $this->getRecords();
-		$tree = array_merge($tree, $treeWithItems);
-
-		return $tree;
+		return array_merge($tree, $treeWithItems);
```
Comment (TomaszQr) [inline modules/Products/models/TreeView.php]: array_merge(parent::getTreeList(), $this->getRecords()) ?

File: modules/Reservations/uitypes/Time.php [hunk 3]
```diff
@@ -42,8 +42,6 @@ public function getDisplayTimeDifferenceValue($fieldName, $value)
 		}
 
 		$dateTimeField = new DateTimeField($date->format('Y-m-d H:i:s'));
-		$value = $dateTimeField->getDisplayTime();
-
-		return $value;
+		return $dateTimeField->getDisplayTime();
```
Comment (TomaszQr) [inline modules/Reservations/uitypes/Time.php]: (new DateTimeField($date->format('Y-m-d H:i:s')))->getDisplayTime()

File: modules/Services/models/TreeView.php [hunk 4]
```diff
@@ -47,8 +47,6 @@ public function getTreeList()
 	{
 		$tree = parent::getTreeList();
 		$treeWithItems = $this->getRecords();
-		$tree = array_merge($tree, $treeWithItems);
-
-		return $tree;
+		return array_merge($tree, $treeWithItems);
```
Comment (TomaszQr) [inline modules/Services/models/TreeView.php]: ?

File: modules/Settings/Countries/models/Module.php [hunk 5]
```diff
@@ -34,11 +34,9 @@ public function updateSequence($sequencesList)
 	public function updateStatus($id, $status)
 	{
 		$db = App\Db::getInstance();
-		$result = $db->createCommand()
+		return $db->createCommand()
```
Comment (TomaszQr) [inline modules/Settings/Countries/models/Module.php]: ?

File: modules/Settings/Countries/models/Module.php [hunk 6]
```diff
@@ -51,11 +49,9 @@ public function updateStatus($id, $status)
 	public function updateAllStatuses($status)
 	{
 		$db = App\Db::getInstance();
-		$result = $db->createCommand()
+		return $db->createCommand()
```
Comment (TomaszQr) [inline modules/Settings/Countries/models/Module.php]: ?

File: modules/Settings/Countries/models/Module.php [hunk 7]
```diff
@@ -69,11 +65,9 @@ public function updateAllStatuses($status)
 	public function updatePhone($id, $phone)
 	{
 		$db = App\Db::getInstance();
-		$result = $db->createCommand()
+		return $db->createCommand()
```
Comment (TomaszQr) [inline modules/Settings/Countries/models/Module.php]: ?

File: modules/Settings/Countries/models/Module.php [hunk 8]
```diff
@@ -87,10 +81,8 @@ public function updatePhone($id, $phone)
 	public function updateUitype($id, $uitype)
 	{
 		$db = App\Db::getInstance();
-		$result = $db->createCommand()
+		return $db->createCommand()
```
Comment (TomaszQr) [inline modules/Settings/Countries/models/Module.php]: ?

File: modules/Settings/CronTasks/models/Record.php [hunk 9]
```diff
@@ -146,8 +146,7 @@ public function getTimeDiff()
 	{
 		$lastStart = (int) ($this->get('laststart'));
 		$lastEnd = (int) ($this->get('lastend'));
-		$timeDiff = $lastEnd - $lastStart;
-		return $timeDiff;
+		return $lastEnd - $lastStart;
```
Comment (TomaszQr) [inline modules/Settings/CronTasks/models/Record.php]: ?

File: modules/Campaigns/models/Module.php [hunk 10]
```diff
@@ -21,9 +21,7 @@ class Campaigns_Module_Model extends Vtiger_Module_Model
 	public function getSpecificRelationQuery($relatedModule)
```
Comment (mariuszkrzaczkowski) [inline modules/Campaigns/models/Module.php]: Function to delete

File: modules/Products/models/Module.php [hunk 11]
```diff
@@ -60,9 +60,7 @@ public function getQueryByModuleField($sourceModule, $field, $record, \App\Query
 	public function getSpecificRelationQuery($relatedModule)
```
Comment (mariuszkrzaczkowski) [inline modules/Products/models/Module.php]: Function to delete

**Pull request merged**